src/components/Footer/index.tsx

```diff
@@ -1,11 +1,11 @@
-import React from "react"
+import * as React from "react"
```

Comment:
Codacy found an issue: ['React' was used before it was defined.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)